packages/http-server/src/http-server.ts

```diff
@@ -4,22 +4,28 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {createServer, Server, ServerRequest, ServerResponse} from 'http';
+import {createServer as createHttpsServer, Server as HttpsServer} from 'https';
```

Comment:
Please use `https.createServer` and `https.Server` instead, see the next line where you are importing everything from `https`.

For consistency, it would be great to do the same for `http.Server` and `http.createServer` (see line 6). I think it makes sense to import `ServerRequest` and `ServerResponse` directly from `http` as you are doing now.